src/app/app.module.ts

```diff
@@ -3,10 +3,11 @@ import { BrowserModule } from '@angular/platform-browser';
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
 import { LayoutModule } from './page/layout/layout.module';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 
 @NgModule({
   declarations: [AppComponent],
-  imports: [BrowserModule, AppRoutingModule, LayoutModule],
+  imports: [BrowserModule, AppRoutingModule, LayoutModule, BrowserAnimationsModule],
```

Comment:
A material module should be included here to import all the modules of Angular Material. We can do it together

Reply:
I created a module file that is in charge of grouping all the Angular Material modules, I just didn't know where to put it so I put it in the root directory

Reply:
Perfect!